spec/std/raise_spec.cr

```diff
@@ -1,16 +1,16 @@
-require "spec"
+require "./spec_helper"
 
 describe "raise" do
   callstack_on_rescue = nil
 
-  it "should set exception's callstack" do
+  pending_win32 "should set exception's callstack" do
```

Comment:
We don't have callstacks on windows right?

Reply:
Not yet, but we should eventually, I suppose?